styles/mixins.less

```diff
@@ -17,7 +17,7 @@
 .agate-item-sizing() {
 	font-size: @agate-item-font-size;
 	font-weight: @agate-item-font-weight;
-	line-height: @agate-item-height;
+	line-height: @agate-item-line-height;
```

Comment:
It seems counter-intuitive, but this was intentionally set as the item-height. This allows the text rendered into the item to be perfectly vertically centered, rather than potentially off-center due to an unintentional mismatch in item height and item line-height.

If your intent was to have the text not be vertically aligned in an item, then this is the correct approach; although you should restore this relationship in the base skin to not break the other skins, and set the mismatched (off-center alignment) values in the skin you want to affect.

```css
// Keep the change above
// Mixins
.agate-item-sizing() {
	...
	line-height: @agate-item-line-height;
	...
}

// Base
@agate-item-line-height: @agate-item-height;

// Silicon override
@agate-item-line-height: 300px;
```